Sources/PortalSwift/Provider/PortalProvider.swift

```diff
@@ -51,80 +54,43 @@ public class PortalProvider {
   ///   - autoApprove: Auto approves all transactions.
```

Comment:
This class is only initialized internally, so updated the initializer to use the new `rpcConfig` model and simplify the argument structure a bit.